library/std/src/sys/unix/process/process_unsupported.rs

```diff
@@ -1,10 +1,8 @@
+#![allow(unused)]
```

Comment:
Why is this necessary? Could it be moved to specific items that may need this?

Reply:
Updated to use a more specific cfg.